examples/text-generation/requirements_awq.txt

```
@@ -1,3 +1,3 @@
 triton==3.1.0
-autoawq
+autoawq==0.2.8
```

```suggestion
autoawq==0.2.7
```

this version seems to work! I will do more test